content/soarian/r4/clinical/summary/condition.md

```diff
@@ -0,0 +1,125 @@
+
+Soarian Clinicals<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` and `POST` based [search] and `GET` based [read] interactions. The response represents the most current information about the patient that is charted in Soarian Clinicals<sup>®</sup> at the time of the query.
+
+The search results include the following fields if they contain values:
+
+* [Id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](https://hl7.org/fhir/R4/condition-definitions.html#Condition.clinicalStatus){:target="_blank"}
+* [Verification Status](https://hl7.org/fhir/R4/condition-definitions.html#Condition.verificationStatus){:target="_blank"}
+* [Category](https://hl7.org/fhir/R4/condition-definitions.html#Condition.category){:target="_blank"}
+* [Code](https://hl7.org/fhir/R4/condition-definitions.html#Condition.code){:target="_blank"}
+* [Subject](https://hl7.org/fhir/R4/condition-definitions.html#Condition.subject){:target="_blank"}
+* [Encounter](https://hl7.org/fhir/R4/condition-definitions.html#Condition.encounter){:target="_blank"}
+  
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_condition, :r4) %>
+
+## Search
+
+Search for Condition resources that meet the specified query parameters:
+
+    GET /Condition?:parameters
+
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true)%>
+
+### Parameters
+
+ Name          | Required?                                    | Type                                                       | Description
+---------------|----------------------------------------------|------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------
+ `_id`         | This, or `patient`                           | [`token`](https://hl7.org/fhir/R4/search.html#token)        | The logical resource id associated with the resource.
+ `patient`	   | This, or `_id`					              | [`reference`](https://hl7.org/fhir/r4/search.html#reference)| The patient who has the condition.
+ `encounter`   | No 										  | [`reference`](https://hl7.org/fhir/r4/search.html#reference)| The encounter associated with the condition.
+ `status`      | No 										  | [`token`](https://hl7.org/fhir/R4/search.html#token)        | The status of the condition. May be a list separated by commas.
+ `_revinclude` | No 										  | [`_revinclude`](https://hl7.org/fhir/search.html#revinclude)| A request to include any Provenance resource in the bundle that refers to condition resource in the search results. Only supported with Provenance. 
```

Comment:
Please add "the" before "condition resource", so the sentence reads as follows:
A request to include any Provenance resource in the bundle that refers to the condition resource in the search results.

Reply:
Done. File has been updated.

Reply:
Looks good, thank you.